channels/email/settings.rst

```diff
@@ -5,101 +5,187 @@ Settings
    :alt: Account settings page
    :align: center
 
-Below you can find the currently available Email related settings. Most of these settings have default values which can be found in this list as well.
+Below you can find the currently available Email related settings. Most of
```

Comment:
sure? `Email related` came from native speaker Ryan.
On a quick search Oracle doesn't seem to use dashes as well

![image](https://user-images.githubusercontent.com/6549061/135463006-bfd4d5c0-cfcb-41f8-88ba-1203d3715d75.png)

Reply:
Both can be correct. Depends on the position in the sentence:
https://english.stackexchange.com/questions/2908/should-i-use-related-or-related

So it would either be "the currently available email-related setting" or "the current settings which are Email related"

I dunno what Oracle based their usage on though... we can also leave it as is, no one will mind